webapp/loggedin/modules/users/user/components/profilePictureEditor.js

```diff
@@ -0,0 +1,121 @@
+import './profilePictureEditor.scss'
+
+import { image as fileTimeImage } from '../../../../../utils/fileTypes'
+
+import React, { useEffect, useRef, useState } from 'react'
+import AvatarEditor from 'react-avatar-editor'
+
+import { useFileDrop } from '../../../../../commonComponents/hooks'
+import useI18n from '../../../../../commonComponents/useI18n'
+
+const ProfilePictureEditor = ({ image: initialImage, onPictureUpdate, enabled }) => {
+
+  const i18n = useI18n()
+
+  const dropRef = useRef(null)
+  const avatarRef = useRef(null)
+
+  const [state, setState] = useState({
+    image: null,
+    scale: 1,
+    rotate: 0,
+  })
+
+  const setScale = scale => setState(statePrev => ({
+    ...statePrev,
+    scale,
+  }))
+
+  const setRotate = rotate => setState(statePrev => ({
+    ...statePrev,
+    rotate,
+  }))
+
+  useEffect(() => {
+    if (initialImage) {
+      setState(statePrev => ({
+        ...statePrev,
+        image: initialImage,
+      }))
+    }
+  }, [initialImage])
+
+  useFileDrop(
+    image => {
+      setState(statePrev => ({
+        ...statePrev,
+        image,
+      }))
+    },
+    dropRef, [fileTimeImage])
+
+  const resetSliders = () => {
+    setState(statePrev => ({
+      ...statePrev,
+      scale: 1,
+      rotate: 0,
+    }))
+  }
+
+  const onImageChange = () => {
+    if (avatarRef.current) {
+      const canvas = avatarRef.current.getImageScaledToCanvas()
+      canvas.toBlob(onPictureUpdate, 'image/jpeg', 0.6)
+    }
+  }
+
+  return (
+    <>
+      <div ref={dropRef} className="profile-picture-editor">
+        { !enabled && <div className="drop-text">{i18n.t('profilePictureEditor.imageDrop')}</div> }
+        {
+          state.image && <AvatarEditor
```

Comment:
when clicking on the editor, the upload picture should appear (in **avatar editor**, in picture show 'Drag & Drop an image here or click to upload', on the right side or below show  zoom, rotate,) see trello: https://trello.com/c/pQLsAnAS/257-feature-user-profile-picture